handlers/officerDeltaHandler.go

```diff
@@ -28,34 +28,39 @@ func NewOfficerDeltaHandler(kSvc services.KafkaService, h helpers.Helper, chv va
 // encountered then they will be returned via the ResponseWriter.
 func (kp *OfficerDeltaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
-	log.Info(fmt.Sprintf("Open API spec to use: %s", kp.cfg.OpenApiSpec), nil)
+	contextId, err := kp.h.GetRequestIdFromHeader(r)
+	if err != nil {
+		log.Error(err)
+	}
+
+	log.InfoC(contextId, fmt.Sprintf("Using the open api spec: "), log.Data{config.OpenApiSpec : kp.cfg.OpenApiSpec})
 
 	// Validate against the open API 3 spec before progressing any further.
-	errValidation, err := kp.chv.ValidateRequestAgainstOpenApiSpec(r, kp.cfg.OpenApiSpec)
+	errValidation, err := kp.chv.ValidateRequestAgainstOpenApiSpec(r, kp.cfg.OpenApiSpec, contextId)
 	if err != nil {
 		w.WriteHeader(http.StatusInternalServerError)
-		log.Error(fmt.Errorf("error occured while trying to validate request: %s", err))
+		log.ErrorC(contextId, fmt.Errorf("error occured while trying to validate request: %s", err))
```

Comment:
format of ErrorC can be 
log.ErrorC(contextId, err, log.Data{"msg": "error occurred while trying to validate request"})

Reply:
typo in `occured`